Add react-add-to-calendar

scottfirestone opened the pull request:
Add to calendar buttons on event rows

This PR closes #334 

#### What does this PR do?
This includes "add to calendar" buttons on the musician and organizer dashboards for each event. Once clicked, the button provides links for adding the event to the user's calendar of choice.
Note 1: "REJECTED"-status events on the musician dashboard will not include this button.
Note 2: We should consider adding a similar "add to calendar" button on event detail pages. To avoid confusing UI, the status (e.g. APPROVED, REJECTED, etc.) will need to be added to the event endpoint.

#### How does this PR make you feel? [:link:](http://gph.is/1yLfi0t)

![](https://media.giphy.com/media/yvk6nmGQ4jEIg/giphy.gif)

scottfirestone committed:
Add react-add-to-calendar

Add to calendar buttons on event rows

nlkluth commented on package.json:
```diff
@@ -78,6 +78,7 @@
     "jest": "^19.0.2",
     "json-loader": "^0.5.4",
     "nsp": "^2.1.0",
+    "react-add-to-calendar": "^0.1.4",
```
Can you move this to `dependencies` instead of `devDependencies`?

nlkluth reviewed (changes requested):
This is pretty cool 👍 - just one comment on the package

scottfirestone commented:
@nlkluth Good call. I made those changes and also fixed a couple linter warnings. 

I messed up the workflow and squashed/rebased the changes, but they should now be up.

fredmon3 committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into 334-export-to-calendar

fredmon3 committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into 334-export-to-calendar

fredmon3 committed:
Fix ESLint failure
